### ISSUE TITLE
Published docker image has no matching manifest for linux/arm64/v8

### ISSUE THREAD

**Issue by stephannielsen** [parse-community/parse-dashboard#2501]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-server/blob/master/SECURITY.md).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/master/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/parse-dashboard/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and [Parse Dashboard](https://github.com/parse-community/parse-dashboard/releases). 

### Issue Description

Starting with release v5.1.0 the published Docker image on Docker Hub no longer has a matching manifest for linux/arm64/v8. When running the Docker image, this message is returned when pulling the image:
```
no matching manifest for linux/arm64/v8 in the manifest list entries
```
With release 5.0.0 there is no problem. Is there a specific reason, why the published image was changed? This prevents us from using the latest version locally (M1 Mac) and in deployments (Linux).

### Steps to reproduce

Pull the image on Linux/Mac M1/ARM.

### Actual Outcome

`no matching manifest for linux/arm64/v8 in the manifest list entries`

### Expected Outcome

Image can be run on linux/arm without issue as before.

### Environment

Starting with v5.1.0 the problem occurs, also with 5.2.0.

Dashboard
- Parse Dashboard version: `> 5.0.0`

Server
- Parse Server version: `not relevant`

Database
- System (MongoDB or Postgres): `not relevant`

### Logs

`no matching manifest for linux/arm64/v8 in the manifest list entries`

**Comment by mtrezza** [comment]
Here is the diff between 5.0.0 and 5.1.0, maybe you can tell from the changes why this occurs:

https://github.com/parse-community/parse-dashboard/compare/5.0.0...5.1.0

**Comment by stephannielsen** [comment]
I already checked but the changes to the image and CI don't seem like they could cause this.

I noticed in the `ci.yml` that platform `linux/arm64/v8` is commented because it breaks the build. 

https://github.com/parse-community/parse-dashboard/blob/86c45d870a56c6366df25aa9eada0b8532bbc6a5/.github/workflows/ci.yml#L79C1-L85C46

But that change was made over 2 years ago. Docker image v5.0.0 was released after that and works fine for me (despite it apparently should not?).

**Comment by stephannielsen** [comment]
@mtrezza maybe #2534 fixes this?

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0-alpha.16](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0-alpha.16)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0-beta.2](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0-beta.2)

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [6.0.0](https://github.com/parse-community/parse-dashboard/releases/tag/6.0.0)
